### DISCUSSION TITLE
Upgrade to PostgreSQL 15

### DISCUSSION THREAD

**Discussion by ldubois** [supabase/supabase#9608] [Feature Requests] [answered]
Some features are very interresting,  like RLS in views. We need a upgrade to PostgreSQL 15 :)

**Comment by MatthieuArzel** [comment]
RSL in views would indeed be great!
I'm strugguling with this right now.

**Comment by GaryAustin1** [comment, marked as answer]
15 is out now.  You need to pause and restore existing free projects to get it.  Paid projects need to contact support.
